roles/csv2/tasks/install_configure_csv2.yaml

```diff
@@ -404,6 +425,7 @@
   set_fact:
     haproxy_name: "{{ 'haproxy.service' if ansible_facts['distribution_major_version'] == '9' else 'HAProxy.service'}}"
 
+#haproxy trying to bind to 0.0.0.0:5000
```

Comment:
remove comment

Reply:
removed in [a8c8e48](https://github.com/hep-gc/uvic-heprc-ansible-playbooks/pull/14/commits/a8c8e481603b07b97218bbb4537fb3acc3130f49)